settings.gradle.kts

```diff
@@ -239,6 +239,9 @@ dependencyResolutionManagement {
       library("protobuf-java", "com.google.protobuf", "protobuf-java")
           .versionRef("protobuf-java-version")
       library("swirlds-common", "com.swirlds", "swirlds-common").versionRef("swirlds-version")
+      library("swirlds-config", "com.swirlds", "swirlds-config").versionRef("swirlds-version")
+      library("swirlds-config-impl", "com.swirlds", "swirlds-config-impl")
```

Comment:
Why do we need `swirlds-config-impl`? Is that a temporary dependency? Otherwise I would say, something is wrong.

Reply:
we need it for tests